extensions/amp-auto-ads/0.1/ad-network-config.js

```diff
@@ -223,6 +226,83 @@ class AdSenseNetworkConfig {
   }
 }
 
+/**
+ * @implements {AdNetworkConfigDef}
+ */
+class DenakopNetworkConfig {
```

Comment:
let's move this into its own file. we should later split all different vendor config to its own files.

Reply:
Would you suggest any filename or location ? I was thinking about `extensions/amp-auto-ads/0.1/ad-denakop-network-config.js` is that fine ?